isis/tests/Fixtures.h

```diff
@@ -121,6 +121,19 @@ namespace Isis {
       void TearDown() override;
   };
 
+  class ApolloNetwork: public TempTestingFiles {
+    protected:
+      Cube *cube1, *cube2, *cube3, *cube4, *cube5, *cube6, *cube7; 
+      FileName *isdFile1, *isdFile2, *isdFile3, *isdFile4, *isdFile5, *isdFile6, *isdFile7, 
+           *cubeLabel, *label1, *label2, *label3, *label4, *label5, *label6, *label7;
+
```

Comment:
Arrays would probably clean some of this up.

Reply:
also, seems like the network and it's path is missing

Reply:
Added the path for the control network. For the network, since it's opened by jigsaw from a file, I didn't include the ControlNetwork itself as a member variable in the Fixture class. Definitely let me know if I missed something here!